Code/Source/RobotControl/RobotControl.h

```diff
@@ -26,15 +31,22 @@ namespace ROS2
     class RobotControl : public IRobotControl
     {
     public:
-        // TODO - pass component args (qos, topic name, etc) as editor-enabled serializable struct
```

Comment:
Also needs a virtual destructor (I know, it was missing it before but lets fix it ;) )